Add troubleshooting step (#7370)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-qs.md

<AboutFusion />

### Troubleshooting

#### How can I address the `dbt language server is not running in this workspace` error

> To avoid the `dbt language server is not running in this workspace` error, you should open your dbt project as a workspace: 

Author: ```suggestion
To avoid the `dbt language server is not running in this workspace` error, you need to add your dbt project folder to a workspace: 
```

> 1. Press CMD+Shift+p and add a folder to your workspace.

Author: ```suggestion
1. In VS Code, click **File** in the toolbar then select **Add Folder to Workspace**
```

> 2. Save this workspace as a file to make it easy to reopen in the future.

Author: ```suggestion
2. To save your workspace click click **File** then select **Save Workspace As**.  Navigate to an accessible location to save your workspace.
3. You can now open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).
```

<AboutFusion />

### Troubleshooting

#### How can I address the `dbt language server is not running in this workspace` error

To avoid the `dbt language server is not running in this workspace` error, you need to add your dbt project folder to a workspace: 

1. In VS Code, click **File** in the toolbar then select **Add Folder to Workspace**

> 2. To save your workspace click click **File** then select **Save Workspace As**.  Navigate to an accessible location to save your workspace.

Author: ```suggestion
2. To save your workspace click **File** then select **Save Workspace As**.  
3. Navigate to the location you want to save your workspace.
```

<AboutFusion />

### Troubleshooting

#### How can I address the `dbt language server is not running in this workspace` error

To avoid the `dbt language server is not running in this workspace` error, you need to add your dbt project folder to a workspace: 

1. In VS Code, click **File** in the toolbar then select **Add Folder to Workspace**
2. To save your workspace click **File** then select **Save Workspace As**.  
3. Navigate to the location you want to save your workspace.

> 3. You can now open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).

Author: ```suggestion
You can now avoid the error and open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).
```

<AboutFusion />

### Troubleshooting

#### How can I address the `dbt language server is not running in this workspace` error

To avoid the `dbt language server is not running in this workspace` error, you need to add your dbt project folder to a workspace: 

> 1. In VS Code, click **File** in the toolbar then select **Add Folder to Workspace**

Reviewer: ```suggestion
1. In VS Code, click **File** in the toolbar then select **Add Folder to Workspace**.
```

2. Select the dbt project file you want to add to a workspace.

> 3. To save your workspace click **File** then select **Save Workspace As**.  

Reviewer: ```suggestion
3. To save your workspace, click **File** then select **Save Workspace As**.  
```

4. Navigate to the location you want to save your workspace.

> You can now avoid the error and open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).

Author: ```suggestion

You can now avoid the error and open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).
```

<AboutFusion />

> ### Troubleshooting

Reviewer: I suggest we add this as a new header after "More information about Fusion" for it to be easier to find?

Author: I think we need to leave it as "Troubleshooting" because we've typically named error message sections this way. Also as far as Google searches, because this is actually treated as one page, I think renaming won't improve the Google search.

Author: Ohhh I see what you mean! Yes we should!

> ### Troubleshooting

Author: ```suggestion
## Troubleshooting
```

> #### How can I address the `dbt language server is not running in this workspace` error

Reviewer 2: I don't think we can make it as short as would be ideal, but shorter may help if it gets linked anywhere
```suggestion
#### `dbt language server is not running in this workspace` error resolution
```

> #### How can I address the `dbt language server is not running in this workspace` error

Author: ```suggestion
#### Addressing the `dbt language server is not running in this workspace` error
```

> To avoid the `dbt language server is not running in this workspace` error, you need to add your dbt project folder to a workspace: 

Reviewer 2: Assuming they're here because they've already encountered it
```suggestion
To resolve the `dbt language server is not running in this workspace` error, you need to add your dbt project folder to a workspace: 
```

1. In VS Code, click **File** in the toolbar then select **Add Folder to Workspace**.
2. Select the dbt project file you want to add to a workspace.
3. To save your workspace, click **File** then select **Save Workspace As**.  
4. Navigate to the location you want to save your workspace.

> You can now avoid the error and open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).

Reviewer 2: ```suggestion
This should resolve the error and open your dbt project by opening the workspace it belongs to. For more information on workspaces, refer to [What is a VS Code workspace?](https://code.visualstudio.com/docs/editing/workspaces/workspaces).
```
